workers/data_refinery_workers/processors/utils.py

```diff
@@ -0,0 +1,72 @@
+import os
+import shutil
+from django.utils import timezone
+from typing import List, Dict, Callable
+from data_refinery_models.models import Batch, BatchStatuses, ProcessorJob
+
+# This path is within the Docker container.
+ROOT_URI = "/home/user/data_store/"
+
+
+def start_job(kwargs: Dict):
+    """Record in the database that this job is being started and
+    retrieve the job and batch from the database."""
+    job = (ProcessorJob
+           .objects
+           .filter(id=kwargs["job_id"])
+           [:1]
+           .get())
+
+    job.worker_id = "For now there's only one. For now..."
+    job.start_time = timezone.now()
+    job.save()
+
+    batch = (Batch
+             .objects
+             .filter(id=job.batch_id)
+             [:1]
+             .get())
+
+    return {"job": job,
+            "batch": batch}
+
+
+def end_job(kwargs: Dict):
+    """Record in the database that this job has completed and that
+    the batch has been processed."""
+    job = kwargs["job"]
+    batch = kwargs["batch"]
+
+    if "success" in kwargs:
+        success = kwargs["success"]
+    else:
+        success = True
+
+    job.success = success
+    job.end_time = timezone.now()
+    job.save()
+
+    batch.status = BatchStatuses.PROCESSED.value
+    batch.save()
+
+    return {}
```

Comment:
It is okay if you want to keep the convention. You can write a brief comment here so that the other readers (and maybe yourself in the future) will understand the reason.